templates/spring-kotlin/files/src/main/webapp/WEB-INF/jsp/index.jsp

```diff
@@ -0,0 +1,86 @@
+<%@taglib uri="http://www.springframework.org/tags" prefix="spring" %>
```

Comment:
Is this file from a standard Kotlin starter kit?

Reply:
This file does not make sense.
I removed it.